habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -640,9 +640,10 @@ def _add_objs(
                 ao = ao_mgr.get_object_by_handle(aoi_handle)
                 if self._kinematic_mode:
                     ao.motion_type = habitat_sim.physics.MotionType.KINEMATIC
-                    # remove any existing motors when converting to kinematic AO
-                    for motor_id in ao.existing_joint_motor_ids:
-                        ao.remove_joint_motor(motor_id)
+                    # TODO: figure out if we should uncomment that @aclegg
```

Comment:
I think Alex recently added this for better performance? Why comment it out?

Reply:
The robot uses the joints to move the arm. If we dont comment out, we get the issue Ruslan was saying, where the joint 20 does not exist.

Reply:
@rpartsey you can verify that this should not create the issue with joint motor 20